src/appendices/spherical-harmonics.md

```diff
@@ -0,0 +1,106 @@
+# Spherical Harmonics
+
+Spherical Harmonics (SH) are continuous functions defined on the surface of the sphere.
+They may be used to encode directionally dependent information;
+for instance, the distribution of white matter fibre orientations within an image voxel.
+This page provides the requisite information on the different basis sets that may be used,
+and how coefficients of such functions may be encoded within a data file.
+
+## SH functions
+
+![SH functions](https://latex.codecogs.com/gif.latex?Y_l^m(\theta,\phi)&space;=&space;\sqrt{\frac{(2l&plus;1)}{4\pi}\frac{(l-m)!}{(l&plus;m)!}}&space;P_l^m(\cos&space;\theta)&space;e^{im\phi}")
+
+for integer *order* *l*, *phase* *m*, associated Legendre polynomials *P*.
+
+## Truncated basis coefficients
+
+![SH basis coefficients](https://latex.codecogs.com/gif.latex?f(\theta,\phi)&space;=&space;\sum_{l=0}^{l_\text{max}}&space;\sum_{m=-l}^{l}&space;c_l^m&space;Y_l^m(\theta,\phi)")
+
+for *maximum* spherical harmonic order *l<sub>max</sub>*.
+
+## Real-valuedness
+
+All functions represented using a spherical harmonics basis are assumed to be real.
+This implies conjugate symmetry;
+that is: *Y*(*l*,-*m*) = *Y*(*l*,*m*)\*,
+where \* denotes the complex conjugate.
+
+## Antipodal symmetry
+
+All functions represented using a spherical harmonics based are assumed to be antipodally symmetric.
+This implies that all basis functions with odd degree are zero.
+
+## Bases
+
+### `mrtrix3`
+
+![MRtrix3 SH basis](https://latex.codecogs.com/gif.latex?Y_{lm}(\theta,\phi)=\begin{Bmatrix}&space;0&\text{if&space;}l\text{&space;is&space;odd},\\&space;\sqrt{2}\times\text{Im}\left[Y_l^{-m}(\theta,\phi)\right]&\text{if&space;}m<0,\\&space;Y_l^0(\theta,\phi)&\text{if&space;}m=0,\\&space;\sqrt{2}\times\text{Re}\left[Y_l^m(\theta,\phi)\right]&\text{if&space;}m>0\\&space;\end{Bmatrix})
+
+### `descoteaux`
+
+![Descoteaux SH basis functions](https://latex.codecogs.com/gif.latex?Y_{lm}(\theta,\phi)=\begin{Bmatrix}&space;0&\text{if&space;}l\text{&space;is&space;odd},\\&space;\sqrt{2}\times\text{Re}\left[Y_l^{-m}(\theta,\phi)\right]&\text{if&space;}m<0,\\&space;Y_l^0(\theta,\phi)&\text{if&space;}m=0,\\&space;\sqrt{2}\times\text{Im}\left[Y_l^m(\theta,\phi)\right]&\text{if&space;}m>0\\&space;\end{Bmatrix})
```

Comment:
And same here, somehow the equation ends up all on one line in the markdown preview.